src/main/webapp/app/exercises/shared/submission-export/submission-export-dialog.component.html

```diff
@@ -0,0 +1,69 @@
+<form *ngIf="!isLoading; else loadingContainer" name="exportForm" role="form" (ngSubmit)="exportSubmissions(exercise.id)" #exportForm="ngForm">
+    <div class="modal-header">
+        <h4 class="modal-title" jhiTranslate="instructorDashboard.exportSubmissions.title">Confirm export operation</h4>
+        <button type="button" class="close" data-dismiss="modal" (click)="clear()">&times;</button>
+    </div>
+    <div class="modal-body">
+        <jhi-alert-error></jhi-alert-error>
+        <p [jhiTranslate]="'instructorDashboard.exportSubmissions.question'" [translateValues]="{ exerciseTitle: exercise.title, courseTitle: exercise.course?.title }">
+            Confirm export
+        </p>
+        <textarea
+            name="studentIds"
+            class="export-textarea"
+            [(ngModel)]="submissionExportOptions.participantIdentifierList"
+            required
+            [disabled]="submissionExportOptions.exportAllParticipants"
+        ></textarea>
+
+        <!--
+        Only show download all checkbox for instructors & admins.
```

Comment:
Should also check this on the server. if submissionExportOptions.exportAllParticipants is true but the user is just TA return forbidden()